test/bin/mdx-test/expect/shell-file-inc/script.sh

```diff
@@ -1 +1 @@
-$ sudo rm -rf /
\ No newline at end of file
+$ sudo chmod a-x /usr
```

Comment:
This is never run but simply included as text. I agree this is a poor choice of command to write though and I'm not sure why we wrote this.